src/index.html

```diff
@@ -1,13 +1,444 @@
+                <div class="testimonial__role">
+                  Senior Vice President
+                </div>
+                <p class="testimonial__about">
+                  My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart
+                </p>
+              </div>
+            </div>
+
+            <div class="testimonials__item grid__item grid__item--9-12">
+              <div class="testimonial">
+                <div class="testimonial__photo-container">
+                  <img
+                  src="./images/people/3.jpg"
+                  alt="Leslie Mckinney"
+                  class="testimonial__photo"
+                  >
+                </div>
+                <h3 class="testimonial__name">
+                  <a href="#" class="link">
+                    Max Henry
+                  </a>
+                </h3>
+                <div class="testimonial__role">
+                  Chief Design Officer
+                </div>
+                <p class="testimonial__about">
+                  When my lovely valley of steam rises and half-day sun is above an impermeable
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contacts" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-2">
+              <div class="contacts__info">
+                <div class="contacts__info-title">
+                  Phone
+                </div>
+
+                <a href="tel:+1 234 5555-55-55" class="contacts__info-link">
+                  +1 234 5555-55-55
+                </a>
+              </div>
+
+              <div class="contacts__info">
+                <div class="contacts__info-title">
+                  Email
+                </div>
+
+                <a href="mailto:hello@miami.com" class="contacts__info-link">
+                  hello@miami.com
+                </a>
+              </div>
+            </div>
+
+            <div class="grid__item grid__item--4-5">
+              <div class="contacts__info contacts__info--last">
+                <div class="contacts__info-title">
+                  Adress
+                </div>
+
+                <a
+                  href="https://goo.gl/maps/z67mgvCo7b2sEopU6"
+                  class="contacts__info-link"
+                  target="_blank"
+                >
+                  400 first ave. n.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN 55401
+                </a>
+              </div>
+            </div>
+
+            <div class="grid__item grid__item--8-12">
+              <form action="https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault" method="get">
```

Comment:
user should stay on your landing on submit form. Post method here is more correctly 

use script to disable reloading when submitting the form and reset form values:

```
function stopDefAction(event) {
  document.getElementById('form').reset();
  event.preventDefault();
}

document.getElementById('form').addEventListener(
  'submit', stopDefAction, false
);
```

```suggestion
              <form action="#" method="post" id="form">
```